picoquic/picoquic_internal.h

```diff
@@ -528,7 +529,7 @@ void picoformat_64(uint8_t* bytes, uint64_t n64);
 size_t picoquic_varint_encode(uint8_t* bytes, size_t max_bytes, uint64_t n64);
 void picoquic_varint_encode_16(uint8_t* bytes, uint16_t n16);
 size_t picoquic_varint_decode(const uint8_t* bytes, size_t max_bytes, uint64_t* n64);
-size_t picoquic_varint_skip(uint8_t* bytes);
+size_t picoquic_varint_skip(const uint8_t* bytes);
```

Comment:
I get that adding const is good hygiene, but do we really need that change in the first package? This PR is too big. Anything that reduces its size would be nice.